solr/test-framework/build.gradle

```diff
@@ -65,16 +65,19 @@ dependencies {
   implementation libs.apache.log4j.api
   implementation libs.apache.log4j.core
   implementation libs.dropwizard.metrics.core
-  implementation libs.dropwizard.metrics.jetty10
+  implementation (libs.dropwizard.metrics.jetty12.ee10){
+    exclude group: 'jakarta.servlet', module: 'jakarta.servlet-api'
```

Comment:
as long as we explicitly depend on `libs.jakarta.servlet.api` below (we do), I wonder do we really need to exclude it here?